languages/kotlin/src/main/cpp/jni_container_node.cpp

```
@@ -83,4 +83,15 @@ JNIEXPORT void JNICALL Java_com_github_kavos113_karin_engine_jni_JniContainerNod
     CHECK_JNI_PTR(containerPtr);
     auto *container = reinterpret_cast<ContainerNode *>(containerPtr);
     container->clearChildren();
+}
+
+JNIEXPORT void JNICALL Java_com_github_kavos113_karin_engine_jni_JniContainerNode_insertChild
+    (JNIEnv *env, jclass cls, jlong containerPtr, jlong childPtr, jint index)
+{
+    CHECK_JNI_PTR(containerPtr);
+    CHECK_JNI_PTR(childPtr);
```

Cppcheck (2.21.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)